airflow/providers/amazon/CHANGELOG.rst

```diff
@@ -24,13 +24,53 @@
 Changelog
 ---------
 
+6.0.0
+.....
+
+Breaking changes
+~~~~~~~~~~~~~~~~
+
 .. warning::
   In this version of provider Amazon S3 Connection (``conn_type="s3"``) removed due to the fact that it was always
   an alias to :ref:`Amazon Web Services Connection <howto/connection:aws>` (``conn_type="aws"``).
   In practice the only impact is you won't be able to ``test`` the connection in the web UI / API.
   In order to restore ability to test connection you need to change connection type from **Amazon S3** (``conn_type="s3"``)
   to **Amazon Web Services** (``conn_type="aws"``) manually.
 
+* ``Remove duplicated connection-type within the provider (#26628)``
```

Comment:
I don't think #26628 - break anything, it just remove duplicate connection types. So for end user it do not affect anyway

Reply:
Moved it to `Misc` section